runtime/westend/src/lib.rs

```diff
@@ -91,7 +91,11 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("westend"),
 	impl_name: create_runtime_str!("parity-westend"),
 	authoring_version: 2,
-	spec_version: 51,
+	// On westend we might deploy multiple runtimes without actually doing a
+	// client release. The first two digits reflect the minor version of the
+	// upcoming client software while the last two can be incremented multiple
+	// times as we go through different runtimes.
+	spec_version: 3100,
```

Comment:
Can you elaborate on this? Why can't we simply increment `spec_version` now and with every "minor" version?

Reply:
Nvm, just saw the internal discussion. Long story short - this will allow us to keep spec versioning of multiple chains match release numbers.

Reply:
I reverted back to v51 and added a note to the wiki page explaining the process.